crates/bevy_animation/src/graph.rs

```diff
@@ -810,12 +812,14 @@ impl AssetLoader for AnimationGraphAssetLoader {
                         MigrationSerializedAnimationClip::Legacy(
                             SerializedAnimationClip::AssetPath(path),
                         ) => {
-                            warn!(
-                                "Loaded an AnimationGraph asset which contains a \
-                                legacy-style SerializedAnimationClip. Please re-save the asset \
-                                using AnimationGraph::save to automatically migrate to the new \
-                                format"
-                            );
+                            if !already_warned {
```

Comment:
I suppose `once!()` would warn only once globally and not once per load ? I.e. it can't be used here if we want to warn for each asset that needs to be migrated.